RecoTracker/TkDetLayers/src/TBLayer.h

```diff
@@ -25,14 +25,14 @@ class TBLayer: public BarrelDetLayer {
 
   // GeometricSearchDet interface
   
-  virtual const std::vector<const GeomDet*>& basicComponents() const final {return theBasicComps;}
+  virtual const std::vector<const GeomDet*>& basicComponents() const {return theBasicComps;}
```

Comment:
if it's in plans, maybe easier to leave as is. 
I don't think the CPU cost is significant from this to the run2 tracking.
Vincenzo, any good guess by how much it would slow down (having in mind all de-finalization here)?